.github/workflows/test-integration.yml

```diff
@@ -9,64 +9,89 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why run the tests on the old Ubuntu LTS?

Reply:
See previous comment